openpype/client/entities.py

```diff
@@ -1367,6 +1368,33 @@ def get_representation_parents(project_name, representation):
     return parents_by_repre_id[repre_id]
 
 
+def get_representation_last_created_time_on_site(
```

Comment:
Point 1: This is not site sync (sync server) specific function (should not be in client code|).
Point 2: Python 2 compatibility breaking code.

Reply:
1. Could you make a conclusion up with @kalisp and tell me what to do in the end? He asked me to put it on the client side: https://github.com/pypeclub/OpenPype/pull/4049#discussion_r1014296215
2. I'll fix the python2

Reply:
> He asked me to put it on the client side:

He was reffering that the code should run only on client side "a process" -> in tray or sync server processes. He didn't said that it should be in client code.

Reply:
Thanks for precising.

Reply:
But I don't understand which difference you make between client and server if you call 'client' what is run on sync server side using REST API (as you're suggesting [here](https://github.com/pypeclub/OpenPype/pull/4049#discussion_r1017676506))

Reply:
There was a miscommunitaion on many fronts. We had an internal discussion about it and made a PR that should fix the issues we had (https://github.com/Tilix4/OpenPype/pull/5). Right now there is missing implementation of one method which will be filled soon.

Reply:
Okay thanks, I've commented it a bit.

Reply:
Yeah, sorry, my bad, I am bit struggling with naming of `openpype.client`.

What I meant was that the method should be on sync server module, not necessarily on Rest API (that was the point I tried to make rather clumsily) , BUT @iLLiCiTiT described me reasons why Rest API is important.

(`ModuleManager.get_module_by_name('sync_server")` doesn't return singleton, there might be multiple instances of SyncModule, only one of those actually has access to `sync_server_thread`. On that thread there might be a timer that is delaying downloading. You need Rest API for instances of module to communicate between each other. Same API might be extended in v4 for additional functionality.)

So after discussion we will be doing some modifications, that you could test and hopefully bringing it home soon :).

Reply:
No problem, we finally found our way out :)

I totally understand the power of the Rest API now and it completely makes sense against my first (inaccurate) proposal.